prbt_hardware_support/test/unittests/unittest_libmodbus_client.cpp

```diff
@@ -34,16 +34,27 @@ namespace pilz_modbus_client_test
 
 using namespace prbt_hardware_support;
 
-constexpr unsigned int DEFAULT_PORT {20500};
+static unsigned int active_port {20500};
 constexpr unsigned int DEFAULT_REGISTER_SIZE {514};
 constexpr unsigned int DEFAULT_WRITE_IDX {512};
 constexpr unsigned int DEFAULT_READ_IDX {77};
 
+class LibModbusClientTest : public testing::Test
+{
+public:
+  void TearDown();
+};
+
+void LibModbusClientTest::TearDown()
+{
+  active_port++;
```

Comment:
I would suggest limiting the range of ports somehow. Otherwise, this thing has the potential to take over all your ports over time :wink: